tests/e2e/cypress/integration/Resources-status/02-actions/index.ts

```diff
@@ -18,15 +18,14 @@ before(() => {
 
 When('I select the acknowledge action on a problematic Resource', () => {
   cy.contains(serviceInAcknowledgementName)
-    .parents('div[role="row"]:first')
+    .parent()
+    .parent()
+    .parent()
+    .parent()
```

Comment:
it's not possible to use a particular selector ?

https://docs.cypress.io/api/commands/parents#Syntax
https://docs.cypress.io/api/commands/parentsuntil#Syntax

Reply:
No

Reply:
ok